app/models/app.rb

```diff
@@ -5,6 +5,11 @@ class App < ApplicationRecord
 
   delegate :full_name, to: :user
 
+  PREFERENCES_FOR_INTERVIEW = [
```

Comment:
@eahanson for some reason these are rendering as:

<img width="534" alt="screenshot 2017-04-06 14 37 54" src="https://cloud.githubusercontent.com/assets/59429/24776667/ac83cf80-1ad6-11e7-9077-e4184d249128.png">